src/app/components/about/about.component.html

```diff
@@ -2,24 +2,25 @@
 	<div class="modal-box">
 		<!-- Header -->
 		<div class="modal-top">
-			<div class="title">About Time of Travel</div>
+			<div class="title">About the Time of Travel application v0.5.0</div>
```

Comment:
Consider dynamically loading the version number from the [config.json (line 2)](https://github.com/USGS-WiM/TravelTime/blob/dev/dist/assets/data/config.json#L2).

Reply:
Great idea. Wish I'd thought of it. ;-)